GirafRest/Models/Many-to-Many Relationships/Activity.cs

```diff
@@ -29,51 +30,65 @@ public class Activity
         /// <summary>
         /// The key of the involved resource.
         /// </summary>
-        [Required]
-        public long PictogramKey { get; set; }
-        /// <summary>
-        /// A reference to the actual resource.
-        /// </summary>
-        [ForeignKey("PictogramKey")]
-        public virtual Pictogram Pictogram { get; set; }
 
-        /// <summary>
-        /// Nullable key for TimerKey
-        /// </summary>
+        public virtual ICollection<PictogramRelation> Pictograms { get; set; }
+        
         public long? TimerKey { get; set; }
 
         /// <summary>
         /// A reference to the actual timer.
-        /// </summary>
+        /// </summary

```

Comment:
```suggestion
        /// </summary>

```
The closing ">" should not be deleted?